src/nvim/main.c

```diff
@@ -677,7 +677,7 @@ void getout(int exitval)
   if (did_emsg) {
     // give the user a chance to read the (error) message
     no_wait_return = FALSE;
-    wait_return(FALSE);
+//  wait_return(FALSE);
```

Comment:
Don't think this was an intentional change - it is still present in the Vim though. Figured I'll port it?

https://github.com/vim/vim/blob/87fd0924e2d85213cc111ee7a5122f92216a37c7/src/main.c#L1664

Reply:
Created https://github.com/vim/vim/pull/9172 to clarify.